app/src/main/java/de/danoeh/antennapod/adapter/NavListAdapter.java

```diff
@@ -296,9 +296,17 @@ private View getSectionDividerView(View convertView, ViewGroup parent) {
                 .getSystemService(Context.LAYOUT_INFLATER_SERVICE);
 
         convertView = inflater.inflate(R.layout.nav_section_item, parent, false);
+        TextView feedsFilteredMsg = convertView.findViewById(R.id.nav_feeds_filtered_message);
 
-        convertView.setEnabled(false);
-        convertView.setOnClickListener(null);
+        if (UserPreferences.getFeedFilter() != UserPreferences.FEED_FILTER_NONE) {
```

Comment:
There is an option to hide the subscriptions list from the sidebar. If it is hidden, the filter text should not be displayed.